src/box/CMakeLists.txt

```diff
@@ -221,6 +224,14 @@ add_library(box STATIC
     lua/watcher.c
     ${bin_sources})
 
+if(ENABLE_AUDIT_LOG)
+    list(APPEND box_sources ${AUDIT_LOG_SOURCES})
```

Comment:
This is something like `${CMAKE_CURRENT_SOURCE_DIR}/../../src/box/audit.c`, correct?

Reply:
It's defined as `${PROJECT_SOURCE_DIR}/src/box/audit.c`, see https://github.com/tarantool/tarantool-ee/commit/e89353998804604c0a7ccf0d5778ab780fae7dd2.